src/commands/furry/e621.js

```diff
@@ -0,0 +1,30 @@
+const { Command, SwitchbladeEmbed, Constants } = require('../../')
+const booru = require('booru')
+
+module.exports = class E621 extends Command {
+  constructor (client) {
+    super(client)
+    this.name = 'e621'
+    this.aliases = ['yiff']
+  }
+
+  async run (message) {
+    message.channel.startTyping()
+    const embed = new SwitchbladeEmbed(message.author)
+    if (message.channel.nsfw) {
+      booru.search('e621.net', ['rating:e'], {limit: 1, random: true})
+        .then(booru.commonfy)
+        .then(images => {
+          for (let image of images) {
+            embed.setImage(image.common.file_url)
+            embed.setDescription('Here\'s your yiff!')
+          }

```

Comment:
why the fuck you're looping this?

Reply:
not a loop totally, if theres an image, it gets only 1, than it sends, but tbh, idk why i put that.

Reply:
also, w/o this the command doesn't works